src/app/views/ChannelsList.svelte

```diff
@@ -77,3 +79,7 @@
     <Content size="lg" class="text-center">No messages found.</Content>
   {/each}
 </FlexColumn>
+
+{#if !$canDecrypt}
+  <ChatEnable />
+{/if}
```

Comment:
For some reason, the modal animation isn't kicking in, maybe because of the history bug mentioned below.

Reply:
For the modal animation to properly animate you need to have it bound to the router navigation. This is how I'm doing it now